jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -51,6 +61,36 @@ def get_server_era(pe_version) {
     }
 }
 
+def get_oss_server_era(oss_version) {
+    // TODO: eventually we will probably want to do something more sophisticated
+    //  here; currently only support 'latest'/'master'/'stable' OSS puppetserver,
+    //  and 'latest' agent
+    if (["latest", "master", "stable"].contains(oss_version)) {
+        return [type: "oss",
+                service_name: "puppetserver",
+                version: oss_version,
+                agent_version: "latest",
+                tk_auth: false,
```

Comment:
Any reason we couldn't just go with `tk-auth: true` here - assuming that we're going to generally be testing more recent versions of Puppet Server that would all support it?